### PR TITLE
BACK-4166: Fix EIP712 schema validator reordering too much, fix Opensea schema order

### PR DESCRIPTION
**Pull request by jnicoulaud-ledger**
https://ledgerhq.atlassian.net/browse/BACK-4166

### CODE REVIEW

File: scripts/validation/validators.py [hunk 1]
```diff
@@ -93,18 +90,9 @@ def eip712_schema_validator(data: str, filename: str) -> Tuple[bool, str]:
                 formatted,
             )
             with open(filename, "w") as f:
-                json.dump(loaded, f, indent=4, sort_keys=True, ensure_ascii=False)
-            return False, "Not formatted corrected"
+                f.write(formatted)
+            return False, "Not formatted, corrected"
```
Comment (adrienlacombe-ledger) [inline scripts/validation/validators.py:94]: not formatted correctly?

Comment (jnicoulaud-ledger) [reply scripts/validation/validators.py:94]: I understood it means "The document was not formatted, therefore it was corrected" :)

**Review by YBadiss — APPROVED** [review]
If the hash is fine, looks good to me 👍

**Review by lambertkevin — APPROVED** [review]
Same as @YBadiss, as long as the hash is correct, it's 🚀 👌

File: scripts/validation/validators.py (hunk 1 above)
Comment (adrienlacombe-ledger) [reply scripts/validation/validators.py:94]: I see :) thank you!

**Pull request merged**